features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -552,6 +552,11 @@
         <DCRM>
             <ApplicationRolePermissionRequiredToView>{{oauth.dcrm.application_role_permission_required_to_view}}</ApplicationRolePermissionRequiredToView>
         </DCRM>
+
+        <!--
+        By enabling this config, additional parameters will be append to OAuth2 error url
```

Comment:
lets mentioned the default value and the current params that will be added if the config is set to true

Reply:
done